frontend/src/components/Table/Table.scss

```diff
@@ -77,10 +78,10 @@
       width: 100%;
       height: 100%;
       align-items: center;
-
-      :nth-child(2) {
-        margin-left: 5px;
-      }
+      justify-content: space-evenly;
+      // :nth-child(2) {
+      //   margin-left: 5px;
+      // }
```

Comment:
can the commented out lines just be deleted?

Reply:
Oooops